### ISSUE TITLE
When `gc deploy` is run an unhelpful error is raised.

### ISSUE THREAD

**Issue by Phenominal** [graphcool/graphcool#895]
When I run `gc deploy` from the CLI on my Macbook Terminal, the following error is raised: 

```
Bundling functions... ⣾
{ file: undefined,
  start: undefined,
  length: undefined,
  messageText: 'Cannot find type definition file for \'fs-extra\'.',
  category: 1,
Bundling functions... !
 ▸    Cannot read property 'getLineAndCharacterOfPosition' of undefined
```

The problem is I have no clue what it is related to, any ideas?

*P.S. I am not using `fs-extra` in any of my code, so I am not sure why it pops up with that message.*

**Comment by marktani** [comment]
* Does this happen with every service definition? If not, please share one for which this happens, preferably in a new GitHub repo.
* What's the output of `graphcool version`?

Might be related to #889.

**Comment by Phenominal** [comment]
Hello, my current version is `graphcool/0.7.0 (darwin-x64) node-v7.8.0`. As for it occurring with every service definition, sorry for my lack of the new terminology, but what exactly do you mean?

**Comment by marktani** [comment]
The set of files that consists of `graphcool.yml` and all files it refers to is the service definition.

**Comment by Phenominal** [comment]
Ok, let me create another project and see if that does it too.

**Comment by Phenominal** [comment]
Ok, I have created another test project and it deployed perfectly fine. I then compared the syntax in all of the files and, to no avail, I see no noticeable differences. Also, when trying to export the problematic service as a zip, the zip is not openable.

**Comment by marktani** [comment]
Ok thanks! Definitely looks like a similar issue as #889.

**Comment by Phenominal** [comment]
I was looking at that and the issue did seem similar. Also, not sure if it is the same or not, but I tried adding the newest `fs-extra` to my `package.json` and then tried re-deploying and it still did not work.

**Comment by Phenominal** [comment]
@marktani I wanted to let you know I solved the issue. The problem was I had another `package.json` in a parent folder that contained `fs-extra` as well. Once I removed it from the parent folder the deploy worked fine.

**Comment by timsuchanek** [comment]
Thanks @Phenominal !  I will enforce the `fs-extra` version in the package.json of the CLI so this can't happen anymore.

**Comment by marktani** [comment]
Thanks a lot for the digging @Phenominal 😄

**Comment by timsuchanek** [comment]
This is fixed and published at `graphcool@0.8.0-alpha.8`
